serverless.yml

```diff
@@ -106,8 +101,6 @@ provider:
     BALANCER: 'https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-v2'
     # Table Defintions
     YIELD_SNAPSHOTS_DATA: ${self:custom.table.YIELD_SNAPSHOTS_DATA}
-    SETT_HISTORIC_DATA: ${self:custom.table.SETT_HISTORIC_DATA}
-    # deprecated
     VAULT_SNAPSHOTS_DATA: ${self:custom.table.VAULT_SNAPSHOTS_DATA}
```

Comment:
would you have to remove VAULT_SNAPSHOTS_DATA as well ?

Reply:
no the vault snapshot data is the current snapshots that serve the data, that comment was incorrect